lib/api/BackbeatAPI.js

```diff
@@ -11,6 +11,7 @@ const BackbeatProducer = require('../BackbeatProducer');
 const Healthcheck = require('./Healthcheck');
 const routes = require('./routes');
 const { redisKeys } = require('../../extensions/replication/constants');
+const client = require('prom-client');
```

Comment:
I would suggest renaming the variable to `promClient` because the context is global to backbeat API.

If prometheus server is not available (and nobody is accessing the `/monitoring` route), can you confirm this does not cause trouble?

A corollary is, how does it behave when `/monitoring` is accessed but the prometheus server is not available? I would expect a kind of HTTP error to be returned.

Reply:
I will change the name of the variable to promClient.
If prometheus server is not available (and nobody is accessing the /monitoring route), can you confirm this does not cause trouble? -> This will not cause any problem.
A corollary is, how does it behave when /monitoring is accessed but the prometheus server is not available? -> We just get an error with code: <Error><Code>InvalidURI